contracts/orderbook/Keepers/OrderKeeperClear.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.8.4;
+import "./OrderBookInterface.sol";
+import "./IUniswapV2Router.sol";
+import "../WrappedToken.sol";
+import "@openzeppelin-4.3.2/token/ERC20/extensions/IERC20Metadata.sol";
+
+contract OrderKeeperClear {
+    address factory;
+
+    constructor(address factoryAddress) {
+        factory = factoryAddress;
+    }
+
+    function checkUpkeep(bytes calldata checkData)
```

Comment:
so probably you will need multiple upkeep jobs in case that check does not fit into single blcok. thats why `checkData` 
needs to be https://github.com/bZxNetwork/liquidation_helper/blob/main/contracts/BzxLiquidateV2Polygon.sol#L234